CONTRIBUTING.md

```diff
@@ -1,45 +1,55 @@
-# Contributing
+# Contributing to this repository
 
-We welcome contributions to this repository from anyone.
+We welcome your contributions! There are multiple ways to contribute.
 
-If you want to submit a pull request to fix a bug or enhance an existing
-feature, please first open an issue and link to that issue when you
-submit your pull request.
+## Opening issues
 
-If you have any questions about a possible submission, feel free to open
-an issue too.
+For bugs or enhancement requests, please file a GitHub issue unless it's
+security related. When filing a bug remember that the better written the bug is,
+the more likely it is to be fixed. If you think you've found a security
+vulnerability, do not raise a GitHub issue and follow the instructions in our

```

Comment:
The Remote Console is supported why can we not provide guidance to the user who encounters a security vulnerability to open an SR with Oracle Support?